plugins/stable/_npe2_readers_guide.md

```diff
@@ -34,7 +34,7 @@ ReaderFunction = Callable[[PathOrPaths], List[LayerData]]
 
 def get_reader(path: PathOrPaths) -> Optional[ReaderFunction]:
     # If we recognize the format, we return the actual reader function
-    if isinstance(path, str) and path.endswith("*.xyz"):
+    if isinstance(path, str) and path.endswith(".xyz"):
```

Comment:
`str.endswith` just does plain string matching not glob-like matching, so using something like this in a plugin doesn't work (unless the filename actually contains an `*` before the `.xyz` I guess?).